src/Chapter1.hs

```diff
@@ -653,8 +682,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit n = if absN < 10 then mod absN 10 else firstDigit (div absN 10)
```

Comment:
If `abs n < 10` you don't need to do `mod` on that 🙂 
So you can simplify it a bit:
```suggestion
firstDigit n = if absN < 10 then absN else firstDigit (div absN 10)
```

Reply:
Oh yes! You are absolutely right. Thank you for your comment :)